bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/telescopePosTableModel/TelescopePosTableModel.java

```diff
@@ -139,7 +139,7 @@ private Row createGhostBaseRow(final GhostAsterism a) {
             // base position corresponds to a target or is set to a sky
             // position, which means checking if a.base (the override) exists.
             if (a.overriddenBase().isDefined())
-                row = new BaseCoordinatesRow(a.overriddenBase().get(), true);
+                row = new BaseCoordinatesRow(a.overriddenBase().get(), true, "Base");
```

Comment:
Maybe we should change it in the `BaseCoordinatesRow` constructor itself.  I don't know when we'd ever want to call that "Sky".

Reply:
https://github.com/gemini-hlsw/ocs/blob/develop/bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/telescopePosTableModel/BaseCoordinatesRow.java#L13